tests/testthat/test-encoding.R

```diff
@@ -0,0 +1,21 @@
+context("non-ASCII characters are handled properly")
+
+test_that("non-ASCII characters are handled properly", {
+  skip_if(.Platform$OS.type != "windows")
```

Comment:
Is there a particular reason not to test on unix? If not, I think we can reduce complexity here and test on all platforms I think.

Reply:
Ah ok, did not see https://github.com/krlmlr/enc/pull/18#discussion_r163591997